src/main/scala/strawman/collection/mutable/ArrayDeque.scala

```diff
@@ -0,0 +1,292 @@
+package scala
+package collection.mutable
```

Comment:
I am fine with 2.13. Happy to make whatever changes necessary for that. I just don't want to wait for 2.14 or Dotty or something.